tests/testthat/test-methods.R

```diff
@@ -152,16 +146,14 @@ test_that("within.list is dispatched", {
 
 test_that("as_tibble.list is _not_ dispatched directly", {
   skip_if_not_installed("tibble")
-  expect_identical(
-    nrow(tibble::as_tibble(lint(text = "a = 1", linters = assignment_linter()))),
-    1L
-  )
+
+  l <- lint(text = "a = 1", linters = assignment_linter())
```

Comment:
Better to unpack this, indeed. But let's avoid the `l` shorthand in committed code. `lint_output` or just `lints`?